package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stampede-worker",
-  "version": "1.3.0",
```

Comment:
@tbranch227 can you revert this change? I'll update the version in a separate commit. I have a workflow for that. Thanks!